src/img/icons.json

```diff
@@ -48,7 +48,12 @@
     "tag",
     "three-dots-row",
     "unlock",
-    "wallet"
+    "wallet",
```

Comment:
A thing on naming icons, which I forgot to mention the first time around. 

Icon names should describe what they are, instead of what we are using them for.

So in your case:
- `expenditure` -> `emoji-bag-money-sign` _(emoji to denote this is a full-color icon, rather than just a wire, single color one)_
- `funds` -> `emoji-money-wings`
- `domains` -> emoji-crane`
- `smite` -> `emoji-fire`
- `advanced` -> `emoji-smiley-nerd`

Basically, the rule of thumb is to describe the icon name as best as you can, so that you can understand how it looks like just by looking at it's name.